rfcs/interop_2023.md

```diff
@@ -0,0 +1,100 @@
+# RFC 116: Interop 2023
+
+## Summary
+
+Interop 2023 is an effort to improve interoperability in areas believed to be important to web developers and users. This is the next iteration of [Interop 2022](./interop_2022.md), an effort which is still ongoing at the time of this RFC.
+
+The focus areas will be selected by a proposals period followed by a proposals review and selection process. The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will manage this process and make decisions based on consensus, see [governance](#governance).
+
+There will be a dashboard at https://wpt.fyi/interop-2023.
+
+## Details
+
+The Interop 2022 [repo](https://github.com/web-platform-tests/interop-2022) and [team](https://github.com/orgs/web-platform-tests/teams/interop-2022) will be renamed to omit the year and used for this effort as well.
+
+### Timeline
+
+During 2022:
+
+- September 15 to October 15: Proposals period.
+- October 16 to October 30: Proposal discussion/refinement. (New proposals no longer accepted.)
+- November 1 to November 30: Proposal selection. (The team must reach consensus no later than November 30.)
+- December 1 to December 16:
+  - Start review of tests. Write needed tests.
+  - Start involving PR teams. Write rough drafts of announcements. Begin creating dashboard.
+
+During 2023:
+
+- January 9 to January 20:
+  - Finalize test selection.
+  - Complete announcement plans. Prepare dashboard for launch.
+- January 23 to January 27: Launch.
+
+If necessary, additional changes to the test lists can be made throughout the year with the consensus of the team, see [updating the tests](#updating-the-tests).
+
+### Governance
+
+The [Interop team](https://github.com/orgs/web-platform-tests/teams/interop) will manage the overall Interop 2023 effort. The team will be formed by renaming the [’22 team](https://github.com/orgs/web-platform-tests/teams/interop-2022).
+
+The team will meet regularly, with agenda issues created in the repo at least 48 hours in advance, and meeting minutes posted in the issue.
+
+The team makes decisions based on consensus, which is defined as support from at least two participating organizations and no opposition.
+
+The Interop 2023 effort is subject to the [WPT code of conduct](https://github.com/web-platform-tests/wpt/blob/master/CODE_OF_CONDUCT.md).
+
+### Proposals
+
+During the proposals period (September 15 to October 15), proposals are submitted as issues in the [Interop repo](https://github.com/web-platform-tests/interop). A proposal should be as specific as reasonably possible, but multiple proposals may be grouped later in the selection process.
+
+There are two kinds of proposals: focus areas where progress is measured using test results, and investigation efforts where progress is measuring against a task list. For both kinds of proposals, finishing them should meaningfully advance the state of interoperability for the web platform.
```

Comment:
One thing that came up last year—but we ultimately decided against—was making something "investigation" like for "implement infrastructure for".

Some of the things that have previously been proposed for next year require infrastructure we don't currently have, so it continues to feel like it would be reasonable?

Reply:
I think adding test infrastructure should be allowed as a proposal. "Meaningfully advance the state of interoperability on the web platform" is already quite a vauge requirement. Maybe for the sake of clarifying the spirit of the rule, we should add some examples of what would be acceptable.

Reply:
I think test infra improvements can be good proposals and can meaningfully advance the state of interop in the same way as clarifying specs or writing tests can.

It is mentioned below, is that enough?

Reply:
It's fine as long as we're not going to end up having language-lawyer arguments about what it means for something to "meaningfully advance the state of interoperability for the web platform" :) Another way of looking at that is I'm not sure that this sentence is adding much value given the following subsections.

Reply:
I suppose I added this to explain the intent/aspiration, and not just have a list of requirements. Ultimately the participants aren't required to give a reason at all for their position on each proposal, so I don't think we'll get to the word lawyering.

The charter would be a good place for some word lawyering though, some principles that we can all agree to.

Reply:
I'm just going to drop this bit and leave it to the charter or the call for proposals:

> For both kinds of proposals, finishing them should meaningfully advance the state of interoperability for the web platform.